doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -53,12 +201,21 @@ Serial LTE Modem (SLM)
     * ``AT#XSOCKETOPT=1,53,`` with ``AT#XSOCKETOPT=1,61,4`` to indicate ``RAI_ONGOING``.
     * ``AT#XSOCKETOPT=1,54,`` with ``AT#XSOCKETOPT=1,61,5`` to indicate ``RAI_WAIT_MORE``.
 
-SUIT DFU for nRF54H20
----------------------
+.. _migration_2.8_required_nrf54h:
+
+nRF54H20
+--------
+
+.. toggle::
+
+  * When using the nRF54H20 DK Engineering B (from v0.8.0 to 0.8.2), you must build samples and applications using the ``nrf54h20dk@0.8.0/nrf54h20/cpuapp`` build target.
+    See the following example::
+
+      west build -p -b nrf54h20dk@0.8.0/nrf54h20/cpuapp -T sample.sysbuild.hello_world.nrf54h20dk_cpuapp_cpurad .
```

Comment:
IMO it would be nice to have the location of that sample. I'm not sure where this sample is available, so I was not able to check that example without searching through the whole nrf repo using the `sample.sysbuild.hello_world.nrf54h20dk_cpuapp_cpurad` pattern.